Bottom: f76e36f1f0061179aa216410cdfc66e7b54b254c
Top:    4a0a4499e2c7177a4b79afb24d2f3b4284421462
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-17 13:11:55 +1300

Refresh of define-stream-result

---

diff --git a/testtools/tests/test_testresult.py b/testtools/tests/test_testresult.py
index 21f15fd..b79f288 100644
--- a/testtools/tests/test_testresult.py
+++ b/testtools/tests/test_testresult.py
@@ -491,7 +491,7 @@ class TestStreamResultContract(object):
         self.addCleanup(result.stopTestRun)
         now = datetime.datetime.now(utc)
         args = [[_u("foo"), s] for s in ['exists', 'inprogress', 'xfail',
-            'uxsuccess', 'finish', 'fail', 'skip']]
+            'uxsuccess', 'success', 'fail', 'skip']]
         inputs = list(dict(
             runnable=False,
             test_tags=set(['quux']),
